themes/src/main/resources/theme/base/login/login-reset-password.ftl

```diff
@@ -9,8 +9,8 @@
                     <label for="username" class="${properties.kcLabelClass!}"><#if !realm.loginWithEmailAllowed>${msg("username")}<#elseif !realm.registrationEmailAsUsername>${msg("usernameOrEmail")}<#else>${msg("email")}</#if></label>
                 </div>
                 <div class="${properties.kcInputWrapperClass!}">
-                    <#if auth?has_content && auth.showUsername()>
-                        <input type="text" id="username" name="username" class="${properties.kcInputClass!}" autofocus value="${auth.attemptedUsername}" aria-invalid="<#if messagesPerField.existsError('username')>true</#if>"/>
+                    <#if auth?has_content>
```

Comment:
Looks like the `has_content` check is now unnecessary. The bean should be always there.

Reply:
I don't know, I tried to do as small change as necessary.

Reply:
Sure, but the `else` is unnecessary as per your changes.